### DISCUSSION TITLE
How to change the installation path on Linux

### DISCUSSION THREAD

**Discussion by vazquezrodjoel** [foldynl/QLog#545] [Q&A]
I'm installing from source code.  How can I change the installation path to /opt/qlog/ when I compile?

Thanks.

**Comment by foldynl** [comment]
https://stackoverflow.com/questions/14639459/how-to-change-qmake-prefix-location

**Comment by vazquezrodjoel** [comment]
Ok, thanks for that.  I added 
    `PREFIX = $$(PREFIX)` 
just before
    `isEmpty(PREFIX) {
     PREFIX = /usr/local
   }`
   
  So I can do 
  `PREFIX=/opt/qlog qmake QLog.pro`
  And it work perfect... 
  
  If there was a better way to do it, let me know.  If not, maybe you could consider adding that line on the QLog.pro file.

Thanks again!

**Comment by vazquezrodjoel** [comment]
Also, to add the folders in /opt to the system PATH variable, I added:
`##Add /opt to path 
    120 PATH=${PATH}:$(find /opt/ -type d -name bin | tr '\n' ':' | sed 's/:$//')` 
to my .bashrc file.

This will add all /opt/*/bin folders.  Like JTDX, FLDIGI, QLog, etc.

**Comment by vazquezrodjoel** [comment]
Also, I wanted to install hamlib on /opt/hamlib, but by doing it qlog could not find the rig.  After reading hamlib instructions several time, I did these steps:
`./configure --prefix=/opt/hamlib`
`make`
`make check`
`sudo make install`

Then, Added these 2 lines to /etc/ld.so.conf.d/libc.conf .  This makes the library path available.
`#Hamlib library path:`
`/opt/hamlib/lib`

then ran
`sudo ldconfig`
and that cache the library location for hamlib, and qlog was happy after that.
